packages/cashier/src/Components/recent-transaction/recent-transaction.jsx

```diff
@@ -0,0 +1,115 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+import { ButtonLink, Text, Icon } from '@deriv/components';
+import { localize, Localize } from '@deriv/translations';
+import { epochToMoment } from '@deriv/shared';
+import { connect } from 'Stores/connect';
+import { getStatus } from '../../Constants/transaction-status';
+
+const RecentTransaction = ({ crypto_transactions, currency, onMount }) => {
+    React.useEffect(() => {
+        onMount();
+    }, []);
+
+    if (!crypto_transactions.length) {
+        return null;
+    }
+
+    let { address_hash, status_code, submit_date, transaction_hash, transaction_type } = crypto_transactions[
+        crypto_transactions.length - 1
+    ];
+
+    submit_date = epochToMoment(submit_date).format('MMM D, YYYY');
+    transaction_type = transaction_type[0].toUpperCase() + transaction_type.slice(1);
+    transaction_hash = transaction_hash
+        ? `${transaction_hash?.substring(0, 4)}....${transaction_hash?.substring(transaction_hash.length - 4)}`
+        : localize('Pending');
+    address_hash = `${address_hash.substring(0, 4)}....${address_hash.substring(address_hash.length - 4)}`;
+    status_code = getStatus(status_code);
+
+    const amount = crypto_transactions[crypto_transactions.length - 1].amount;
+
+    return (
+        <div className='cashier-recent-transaction-wrapper'>
+            <div className='cashier-recent-transaction'>
+                <Text weight='bold' as='p' line_height='s' size='xs'>
+                    <Localize i18n_default_text='Recent Transactions' />
+                </Text>
+                <div className='cashier-recent-transaction__data-wrapper'>
+                    <Icon
+                        className='cashier-recent-transaction__icon'
+                        icon={transaction_type === 'deposit' ? 'IcCashierAdd' : 'IcCashierMinus'}
+                        size={32}
+                    />
+                    <div>
+                        <div className='cashier-recent-transaction__status-wrapper'>
+                            <Text as='p' size='xxs'>
+                                {`${transaction_type} ${currency}`}
+                            </Text>
+                            <div className='cashier-recent-transaction__status'>
+                                <span
+                                    className={classNames('cashier-recent-transaction__status-indicator', {
+                                        'cashier-recent-transaction__status-indicator--green':
+                                            status_code === 'Successful',
```

Comment:
I don't think this line works in any language other than English, since the `status_code` is a translated result.

Reply:
is there a way to handle it?

Reply:
You can use the code itself, for example for `Successful` the code is `SENT` or `CONFIRMED`

Reply:
I moved Localize instead :)